README.md

```diff
@@ -27,7 +27,7 @@ The software has been updated in 2020 for current compilers.
 Requirements
 ------------
 
-| Software        | Minimum version | Tested version | Required? | Web site                              |
+| Software        | Minimum version | Tested version | Required? | Website                               |
```

Comment:
All looks fine, except "Website".  Sorry, but I prefer "Web site", unlike most people.  Likewise, I don't like "dataset" as one word.  But that's just me...

I'm new to this and it seems like I can't accept some changes and reject one.  Instead, I think I'm supposed to give you feedback (like this message) and you submit another PR.  Let me know if you can do that or I can just accept it and change it back.  Thank you; everything else is great and thank you!!

Reply:
Good to know thanks, I'll keep in mind in future :)
Change reverted

Reply:
Thanks a lot for the updated PR and for being an extra pair of eyes!